lib/fs/basicfs_windows.go

```diff
@@ -150,3 +151,38 @@ func (f *BasicFilesystem) Roots() ([]string, error) {
 
 	return drives, nil
 }
+
+func (f *BasicFilesystem) resolveWin83(absPath string) string {
+	if !strings.Contains(absPath, "~") {
+		return absPath
+	}
```

Comment:
Perhaps we should check for caps and length? Our own filenames contain ~ on wondows.

Reply:
I made a check that filters out "false positives" due to `~syncthing~`. If you know of a simpler or more precise criterion based on caps/length for that 8.3 stuff, please let me know.